pkg/aws/client.go

```diff
@@ -984,6 +986,27 @@ func (c *awsClient) CreateS3Bucket(bucketName string, region string) error {
 	if err != nil {
 		return err
 	}
+
+	_, err = c.s3Client.DeletePublicAccessBlock(&s3.DeletePublicAccessBlockInput{
+		Bucket: aws.String(bucketName),
+	})
+	if err != nil {
+		return err
+	}
+
+	_, err = c.s3Client.PutBucketOwnershipControls(&s3.PutBucketOwnershipControlsInput{
+		Bucket: aws.String(bucketName),
+		OwnershipControls: &s3.OwnershipControls{
+			Rules: []*s3.OwnershipControlsRule{
+				{
+					ObjectOwnership: aws.String(OwnershipControlBucketOwnerPreferred),
+				},
+			},
+		},
+	})
```

Comment:
We can leave the default object ownership (which is recommended by AWS) and instead setup an object policy

See https://github.com/openshift/cloud-credential-operator/pull/526 for an example.

Reply:
Yes, I got through that option as well. I'm just thinking to provide a temporary solution while we work things out to have it managed within repo (https://github.com/openshift/managed-cluster-config/tree/master/resources/sts)

Reply:
@jharrington22 Let me know what you think so we can move this hardcoded solution temporarily and in the meantime work on the policy repo